src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -137,6 +143,64 @@ public static String extractVersionNumber(String inputString) {
         return inputString.substring(startPos, endPos).replace("-", ".").trim();
     }
     
-    
+    public void verifyReadyForGodMode() {
+        if (!inst().isDevServer()) {
+            Assumption.fail("God mode regeneration works only in dev server.");
+        }
+        if (areTestAccountsDefaultValues()) {
+            Assumption.fail("Please change ALL the default accounts in test.properties in order to use God mode,"
+                            + "e.g change test.student1.account from alice.tmms to alice.tmms.example");
```

Comment:
Something like `alice.tmms.yourName`? I think it could work.
Alternatively, we can hardcode the names that can be used for godmode in `test.template.properties`. The check will then be a strict equality check. This also can solve the problem.